ui/react-components/components/EditAppointment/UpdateButtons.module.scss

```diff
@@ -56,7 +56,7 @@
     color: $secondary-color-1;
     background: $primary-color-2;
     &:hover {
-      background: $primary-color-2;
+      background: #454b86;
```

Comment:
should we check with UX Rohit to get the variable name for this new color

Reply:
Previously on icon of removing ProviderTag also, we added new color on hover. Here also for hovering effect, we added new color code. These(hover effects) are nowhere mentioned in zeplin.